sbin/common/sbom.sh

```diff
@@ -1,4 +1,32 @@
 #!/bin/bash
+# Build the CycloneDX Java library and app used for SBoM generation
+buildCyclonedxLib() {
+  local javaHome="${1}"
+
+  # Check if JDK-17 is available, download it if it isn't
```

Comment:
Move this boot jdk logic to https://github.com/adoptium/temurin-build/blob/18f343c90c72084cad7f6d29f14721e1309ed1ac/sbin/build.sh#L688

Reply:
agree, i was about to say the logic should be in `setupAntEnv() `